application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -141,4 +141,13 @@ public static String abbreviate(String text, int maxLength) {
         return text.substring(0, maxLength - ABBREVIATION.length()) + ABBREVIATION;
     }
 
+    /**
+     * Mentions a guild channel by its id
+     *
+     * @param channelId the channel id to mention
```

Comment:
thats wrong english. instead `@param channelID the ID of the channel to mention`

Reply:
done